pkg/util/cmapichecker/cmapichecker_test.go

```diff
@@ -70,22 +69,11 @@ func TestCheck(t *testing.T) {
 
 	tests := map[string]testT{
 		"no errors": {},
-		"without any cert-manager CRDs installed (missing from /apis)": {
-			apisResponse: func(t *testing.T, r *http.Request) (int, []byte) {
-				return http.StatusOK, []byte(`{
-					"kind": "APIGroupList",
-					"apiVersion": "v1",
-					"groups": []
-				}`)
-			},
-			expectedError:       `error finding the scope of the object: failed to get restmapping: no matches for kind "CertificateRequest" in group "cert-manager.io"`,
-			expectedSimpleError: ErrCertManagerCRDsNotFound.Error(),
-		},
 		"without any cert-manager CRDs installed (404)": {
 			discoveryResponse: func(t *testing.T, r *http.Request) (int, []byte) {
 				return http.StatusNotFound, nil
 			},
-			expectedError:       `error finding the scope of the object: failed to get restmapping: no matches for kind "CertificateRequest" in group "cert-manager.io"`,
+			expectedError:       `error finding the scope of the object: failed to get restmapping: no matches for kind "CertificateRequest" in version "cert-manager.io/v1"`,
```

Comment:
c/r optimized their RESTMapper, resulting in slightly different errors and behavior.